docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -237,7 +237,7 @@ WantedBy=default.target
 
 Podman-generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of systemctl(1) during installation.
 
-Once you have generated the systemd unit file, you can copy the generated systemd file to ```/etc/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user``` for installing it as a non-root user. Enable the copied unit file or files using `systemctl enable`.
+Once the systemd unit file is generated, copy it to ```/etc/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user``` for installing it as a non-root user. Enable the copied unit file or files using `systemctl enable`.
```

Comment:
Fault is in the original wording, but `and` should be **or** ("...or to ... for non-root").